format/format.go

```diff
@@ -133,7 +133,7 @@ func formatValue(value reflect.Value, indentation uint) string {
 	case reflect.Uintptr:
 		return fmt.Sprintf("0x%x", value.Uint())
 	case reflect.Float32, reflect.Float64:
-		return fmt.Sprintf("%v", value.Float())
+		return fmt.Sprintf("%f", value.Float())
```

Comment:
Interesting, didn't realize that. I guess thats why it was %v to begin with.

I guess there should be separate specs for floats with decimal places and the one without